### DISCUSSION TITLE
soundcloud issues

### DISCUSSION THREAD

**Discussion by gazzawazza** [aclap-dev/video-downloadhelper#1504] [Help and Q&A]
### Your question

hi

having an issue here:

https://soundcloud.com/yotto/just-over-extended-mix

Produces two MP4 options (see attachment)

One is the correct length but there's no audio on playback in VLC

Extracting the audio stream (using WinFF and FFMPEG) to produce an M4A just produces a string of errors in foobar ("unsupported format or corrupted file") for pretty much every frame (before foobar gives up lol)

Both this audio extraction method (WinFF to produce M4A from MP4) and using video downloadhelper has historically worked fine on soundcloud

Even if people can reproduce, that would be helpful

Thanks for any assistance

### Addon & CoApp version and details

VDH 9.1.0.44
companion app 2.0.19
firefox 133

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

![#01](https://github.com/user-attachments/assets/3eb5587d-8e0d-4702-813b-28f1db734d84)

Give up.  The content is protected by DRM.  Maybe it was the case that this content was not previously protected by DRM.  But it is now.  That's the sad reality of the present state of affairs.  You are plain flat out of luck.  Give up.

You need to look for that icon on each item you are interested in.  It could be that some content has it & some does not.  Whatever content does not show that icon, you should be able to download that.  But if you see that icon, give up.  This is on Firefox.  It is my understanding that the other browsers are not this helpful.  In that case, you might consider using Firefox instead.

**Reply by GrampaWildWilly** [reply]
Yes, I know you are using Firefox.  But not everybody reading this uses Firefox so that's why I said that.

**Reply by gazzawazza** [reply]
@GrampaWildWilly  thanks for taking the time to reply.

hadn't even noticed that address bar icon.

will look out for it in future and see if it correlates with VDH issues
